Packages/Newtonsoft.Json-for-Unity.Converters/Configuration/KeyedConfig.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace Newtonsoft.Json.UnityConverters.Configuration
+{
+    [Serializable]
+    public struct KeyedConfig
+    {
+        public string key;
+
+        public ConfigType type;
+
+        public bool boolean;
```

Comment:
Codacy found an issue: [Make this field 'private' and encapsulate it in a 'public' property.](https://app.codacy.com/manual/jilleJr/Newtonsoft.Json-for-Unity.Converters/pullRequest?prid=5865055)